resources/metadata/shellExecute.yaml

```diff
@@ -0,0 +1,32 @@
+metadata:
+  name: shellExecute
+  description: Step executes defined script
+  longDescription: Step executes defined script with Vault credentials, or created them on this step
+spec:
+  inputs:
+    params:
+      - name: vaultServerUrl
+        type: string
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        description: The URL for the Vault server to use
+        mandatory: true
+      - name: vaultNamespace
+        type: string
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        description: The vault namespace that should be used (optional)
+        default: true
```

Comment:
`default: true` does not look right.

Default needs to correspond to the type of the parameter.

Reply:
my inattention, thanks, removed